data/campaigns/Northern_Rebirth/scenarios/01_Breaking_the_Chains.cfg

```
@@ -482,19 +482,19 @@
             message= _ "Mission? What mission?"
         [/message]
     [/event]
-    
+
     [event]
         name=time over
         [message]
             speaker= narrator
-            message= _ "Sounds of drums and horns echoes on the side of the mountains... Tallin knew what this sound means: it is too late! A large orc army is coming this way..."
+            message= _ "Sounds of drums and horns echoes on the side of the mountains... Tallin knew what this sound meant: it is too late! A large orc army is coming this way..."
```

I'd suggest avoiding too much extra punctuation unless there's a purpose to having it/you're trying to imply something with it. It makes the sentences hard to read. So, for example:

```suggestion
            message= _ "Sounds of drums and horns echo on the mountainsides. A large orc army draws near!"
```